### PR TITLE
Ramp Model Schemas & Models

### PR DESCRIPTION
**Pull request by PaulHuwe**
Implemented ramp models & schemas. 
Moved group table from level1.schema to group.schema. 
Fixed JWST->Roman wording on core.schema.

### CODE REVIEW

File: romancal/datamodels/ramp.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+from .core import RampDataModel
+#from ..lib.basic_utils import deprecate_class
+
+class RampModel(RomanDataModel):
+    """
+    A data model for 4D ramps.
```
Comment (dmggh) [inline romancal/datamodels/ramp.py:6]: Shouldn't this be for 3D ramps, as there will only be 1 integration ?

Comment (PaulHuwe) [reply romancal/datamodels/ramp.py:5]: Yup.. and I need to fix it everywhere.

Comment (PaulHuwe) [reply romancal/datamodels/ramp.py:5]: Fixed.

File: romancal/datamodels/ramp.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+from .core import RomanDataModel
+
+class RampModel(RomanDataModel):
+    """
+    A data model for 4D ramps.
```
Comment (ddavis-stsci) [inline romancal/datamodels/ramp.py:5]: 4D -> 3D

File: romancal/datamodels/ramp.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+         Error array
+
+    zeroframe : numpy float32 array
+         Zeroframe array
+
+    group : numpy table
+         group parameters table
+
+    int_times : numpy table
+         table of times for each integration
```
Comment (ddavis-stsci) [inline romancal/datamodels/ramp.py:28]: With one integration does this need to be a table?

File: romancal/datamodels/rampfitoutput.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
+from .core import RomanDataModel
+
+class RampFitOutputModel(RomanDataModel):
+    """
+    A data model for the optional output of the ramp fitting step.
+
+    In the parameter definitions below, `n_int` is the number of
+    integrations, `max_seg` is the maximum number of segments that
```
Comment (ddavis-stsci) [inline romancal/datamodels/rampfitoutput.py:8]: remove `n_int` here and below.

File: romancal/datamodels/ramp.py (hunk 3 above)
Comment (PaulHuwe) [reply romancal/datamodels/ramp.py:28]: Good catch, this was actually removed - as it was for others. Deleted.

File: romancal/datamodels/ramp.py (hunk 2 above)
Comment (PaulHuwe) [reply romancal/datamodels/ramp.py:5]: Fixed.

File: romancal/datamodels/rampfitoutput.py (hunk 4 above)
Comment (PaulHuwe) [reply romancal/datamodels/rampfitoutput.py:8]: Fixed this as well as mentions of nread.

**Review by ddavis-stsci — APPROVED** [review]
LGTM

File: romancal/datamodels/__init__.py [hunk 5]
```diff
@@ -22,4 +22,6 @@
 
            "ImageModel",
 
+           "RampModel", "GLS_RampFitModel", "RampFitOutputModel",
```
Comment (nden) [inline romancal/datamodels/__init__.py:25]: These need to be imported from the respective modules in order to be able to define them here.
I'll fix it in #115

**Pull request merged**